packages/expect/src/jasmineUtils.ts

```diff
@@ -193,47 +182,20 @@ function eq(
   return result;
 }
 
-function keys(
-  obj: object,
-  isArray: boolean,
-  hasKey: (obj: object, key: string) => boolean,
-) {
-  var allKeys = (function(o) {
-    var keys = [];
-    for (var key in o) {
-      if (hasKey(o, key)) {
-        keys.push(key);
-      }
+function keys(obj: object, hasKey: (obj: object, key: string) => boolean) {
+  var keys = [];
+  for (var key in obj) {
+    if (hasKey(obj, key)) {
+      keys.push(key);
     }
-    return keys.concat(
-      (Object.getOwnPropertySymbols(o) as Array<any>).filter(
-        symbol =>
-          (Object.getOwnPropertyDescriptor(o, symbol) as PropertyDescriptor)
-            .enumerable,
-      ),
-    );
-  })(obj);
-
-  if (!isArray) {
-    return allKeys;
   }
-
-  var extraKeys = [];
-  if (allKeys.length === 0) {
-    return allKeys;
-  }
-
-  for (var x = 0; x < allKeys.length; x++) {
-    if (
-      typeof allKeys[x] === 'symbol' ||
-      !allKeys[x].match(/^[0-9]+$/) ||
-      Number(allKeys[x]) >= 4294967295
-    ) {
-      extraKeys.push(allKeys[x]);
-    }
-  }
-
-  return extraKeys;
+  return keys.concat(
+    (Object.getOwnPropertySymbols(obj) as Array<any>).filter(
+      symbol =>
+        (Object.getOwnPropertyDescriptor(obj, symbol) as PropertyDescriptor)
+          .enumerable,
```

Comment:
thanks prettier

Reply:
I blame the people who put enumerable symbol-keyed properties on objects in the first place ^^